samples/runtime/autosubstitution/AutoSubstitution.qs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Project {
```

Comment:
Generally, samples should still have namespace names starting with Microsoft.Quantum.